index.js

```diff
@@ -185,10 +185,6 @@ HtmlWebpackPlugin.prototype.evaluateCompilationResult = function(compilation, co
   if(!compilationResult) {
     return Promise.reject('The child compilation didn\'t provide a result');
   }
-  if(typeof source === 'string') {
```

Comment:
Ah sorry, did not know that. I ran the code through Istanbul and found out that this line was not tested. I couldn't find a usecase for it so I removed it. Could you give an example of when this would be used? Then I can write a test for it.